.gitignore

```diff
@@ -4,5 +4,9 @@
 .Ruserdata
 *.nc
 03_historical_jread/*.remake
-.Rproj
+*.Rproj
 .DS_Store
+*.ai
+*.gif
+*/_targets/
+*/scratch.R
```

Comment:
Since the pipeline creates `in/state_water_area.csv` and `out/water_rank.png`, I don't think those should be committed to the repo. They are artifacts of the pipeline, so when people build them, they will be available locally.

Reply:
I chose to commit the csv because it's scraped from WSS, so it would still exist if that page ever changed. It's also very small.